LibCarla/source/carla/Exception.cpp

```diff
@@ -14,14 +14,14 @@
 #ifdef BOOST_NO_EXCEPTIONS
 
 namespace boost {
-
+  
   void throw_exception(const std::exception &e) {
     carla::throw_exception(e);
   }
 
   void throw_exception(
       const std::exception &e,
-      boost::source_location const & loc) {
+      boost::source_location const & ) {
```

Comment:
Return back parameter name

Reply:
done and added (void)loc as the second option to get rid of the compiler warning on this.